From 6f5f78ddddd00f0cf4feab7a73f85eb4cc88932d Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Thu, 23 Mar 2023 10:42:45 +0100
Subject: [PATCH 30/32] GOOD-ENH: Add installation of generated Qt header files

This adds the installation of the `ui_*.h` files as development files (headers)
---
 CMake/SlicerMacroBuildModuleQtLibrary.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/SlicerMacroBuildModuleQtLibrary.cmake b/CMake/SlicerMacroBuildModuleQtLibrary.cmake
index e552e656bf..4914e170f8 100644
--- a/CMake/SlicerMacroBuildModuleQtLibrary.cmake
+++ b/CMake/SlicerMacroBuildModuleQtLibrary.cmake
@@ -211,6 +211,7 @@ macro(SlicerMacroBuildModuleQtLibrary)
     file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
     install(FILES
       ${headers}
+      ${MODULEQTLIBRARY_UI_CXX}
       ${dynamicHeaders}
       DESTINATION ${Slicer_INSTALL_QTLOADABLEMODULES_INCLUDE_DIR}/${MODULEQTLIBRARY_NAME} COMPONENT Development
       )
-- 
2.39.2

